README.md

```diff
@@ -1 +1 @@
-# review-project
\ No newline at end of file
+# The purpose of this project is to look at my previous project and act lika a professional code reviewer.
```

Comment:
Your README file should always be enough so that a technical and non-technical can easily what you really did in this project. So I would highly recommend documenting your project.

Reply:
Great job here for pointing out the unprofessional README file however the comment lacks clarity. Be more specific about what you're looking for in the README file. What information should be included? For example, you could say something like:

"The README file should be more descriptive and professional. It should include a brief overview of the project, as well as installation instructions, usage examples, and a list of contributors. You can this README template: [template](https://github.com/microverseinc/readme-template) for better structured and professional README"

Reply:
- [Optional] Great job for updating based on the previous code reviewer 👍 , I think it would be better if you use a couple of emojis and/or gifs in the review. They make the code review more human-friendly 👽🤖.